docs/source/eventing/actions/label.rst

```diff
@@ -43,14 +43,9 @@ Changes the text and color of a generic label device.
 +----------+------------------+---------------------+---------------------------------------------------+
 | **Name** | **Required**     | **Multiple Values** | **Description**                                   |
 +----------+------------------+---------------------+---------------------------------------------------+
-| BLINK    | |no|             | |no|                | Blinking label text on or off                     |
+| BLINK    | |no|             | |no|                | Blinking label text on OR off                     |
```

Comment:
No, i meant accentuating the on and of like this `on` or `off`

Reply:
Hm.. on my laptop screen that highlighting is hardly visible, so I did not see what you meant.

Done.